### DISCUSSION TITLE
Mengubah format country, agar bisa tau negara dengan jelas (tidak singkatan)

### DISCUSSION THREAD

**Discussion by jeremiaww** [teamalgoritma/community#84] [Q&A] [answered]
Hallo,

Saya ingin bertanya mengenai cara mengubah format country,
Jenis format yang berasal dari dataset : Country of origin. Categories are represented in the ISO 3155–3:2013 format

Yang diinginkan adalah format jenis country seperti biasanya, tidak disingkat.

Dataset yang digunakan berasal dari https://www.kaggle.com/datasets/jessemostipak/hotel-booking-demand?select=hotel_bookings.csv

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/99194718/167247836-f77c09bd-949c-4d7f-8338-b871583f9e8d.png">

Terima kasih

**Comment by VictorNugraha** [comment, marked as answer]
Halo, @jeremiaww !

Untuk mengubah format penamaan kolom _country_ pada dataframe yang digunakan sehingga tidak dalam format singkatan, Bapak bisa memanfaatkan fungsi _**countrycode()**_ dari **_library(countrycode)_**.

Pada fungsi tersebut terdapat 3 parameter yang dapat dimanfaatkan, yaitu:
- _sourcevar_ -> parameter ini akan di-isi dengan nama kolom yang ingin kita ubah, dalam kasus ini kolom _country_.
- _origin_ -> parameter ini akan di-isi dengan format awal penulisan negara pada kolom _country_, dalam kasus ini bisa di-isi dengan **"iso3c"**.
- _destination_ -> parameter ini di-isi dengan format yang diharapkan, dalam kasus ini bisa di-isi dengan **"country.name"**.

Untuk dokumentasi lengkap mengenai library tersebut, Bapak bisa melihat [GitHub](https://github.com/vincentarelbundock/countrycode) dari pembuat library tersebut atau melihat dokumentasi [CRAN](https://cran.r-project.org/web/packages/countrycode/countrycode.pdf) yang disediakan oleh R.

Semoga fungsi tersebut dapat membantu dan selamat mencoba! :D

**Reply by jeremiaww** [reply]
Terima kasih mas atas bantuannya
